Code/Framework/AzCore/AzCore/DOM/DomUtils.cpp

```diff
@@ -123,9 +123,74 @@ namespace AZ::Dom::Utils
         }
     }
 
+    bool CanLoadViaJsonSerialization(
+        const AZ::TypeId& typeId, const Value& root, JsonDeserializerSettings settings)
+    {
+        // A serializeContext is required for making the temporary AZStd::any for storage
+        // so if the supplied serialize context is nullptr, query the one associated with the ComponentApplication
+        auto componentApplicationInterface = AZ::Interface<AZ::ComponentApplicationRequests>::Get();
+        if (settings.m_serializeContext == nullptr)
+        {
+            settings.m_serializeContext = componentApplicationInterface != nullptr ? componentApplicationInterface->GetSerializeContext() : nullptr;
+        }
+        if (settings.m_serializeContext == nullptr)
+        {
+            return false;
+        }
+
+        if (settings.m_registrationContext == nullptr)
+        {
+            settings.m_registrationContext = componentApplicationInterface != nullptr ? componentApplicationInterface->GetJsonRegistrationContext() : nullptr;
+        }
+        if (settings.m_registrationContext == nullptr)
+        {
+            false;
```

Comment:
This should be a return statement.
```suggestion
            return false;
```